shoop_tests/xtheme/test_editor_view.py

```diff
@@ -133,3 +133,20 @@ def test_editor_view_unknown_command():
         view_obj.request.POST = {"command": printable_gibberish()}
         with pytest.raises(Problem):
             view_obj.dispatch(view_obj.request)
+
+
+@pytest.mark.django_db
+def test_editor_cell_limits():
+    layout, svc = get_test_layout_and_svc()
+    cell_limit = ROW_CELL_LIMIT
```

Comment:
No need to make this a local variable (especially since you're using both the local variable and the imported name in the test itself).